en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -15,12 +15,13 @@ We'll go through each of these in detail in the following sections.
 | Main interface | Function |
 | :--- | :--- |
 | Power | Connect Power module;  power input & AD voltage  and current detection. |
-| PM2 | Connect i2c smart battery. |
-| TF CARD | Insert SD card for log storage |
+| PM2 | [Do not use with PX4](../flight_controller/cuav_v5_nano.md#issue_pm2) |
+| TF CARD | SD card for log storage (comes with card) |
 | M1~M8 | PWM outputs. Can be used to control motors or servos. |
-| A1~A8 | PWM outputs. Can be used to control motors or servos. |
+| A1~A3 | PWM outputs. Can be used to control motors or servos. |
+| nARMED | |
```

Comment:
@davids5 @cuhome What is nARMED for?

Reply:
YES, v5 nano only has A1~A3, it can be used as pwm output port, and there is a nARMED pin header.

Reply:
@hamishwillee see https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165&range=A131:G131

Reply:
@cuhome So @davids5 says A1-A3 cannot be used as pwm output port (they are capture pins). So that is what I have said.

Re nARMED, I have documented as:
```
| nARMED | Digital OUTPUT: GPIO will be set as input while not armed. While armed it will be configured as a GPIO OUT set low. |
```
What would you really use this for? Ie when do you need an input before being armed and an output after being amed?

Reply:
I hate how git repeats notes so you have to resolve them twice. FYI David, this is where I asked you about the purpose of nARMED.